tests/exporters/onnx/test_exporters_onnx_cli.py

```diff
@@ -54,6 +54,10 @@ def _get_models_to_test(export_models_dict: Dict):
 
             for model_name, tasks in model_tasks.items():
                 for task in tasks:
+                    if model_type == "encoder-decoder" and task == "seq2seq-lm-with-past":
+                        # The model uses bert as decoder and does not support past key values
+                        continue
```

Comment:
Not sure I understand this. This means that pkv is not tested, but may be supported depending on which arch is used as decoder?

Reply:
yes